plugins/opds.koplugin/opdsbrowser.lua

```diff
@@ -31,32 +31,35 @@ local CatalogCache = Cache:new{
 }
 
 local OPDSBrowser = Menu:extend{
-    opds_servers = G_reader_settings:readSetting("opds_servers", {
+    opds_servers = G_reader_settings:readSetting(
+        "opds_servers",
         {
-            title = "Project Gutenberg",
```

Comment:
These changes add a lot of extra lines to the diff that aren't really necessary. :-/

Reply:
Aw geez, my bad. I was just trying to make the code a bit more readable along the way.

Reply:
> The logic lgtm, but the formatting changes range from a bit unnecessary to probably against our general style.

I agree, that's the first thing that stroke me when having a look at the code.
We are all used to see it like this:
```lua
                        table.insert(item.acquisitions, {
                            type = link.title,
                            href = build_href(href),
                        })
```

Reply:
If that's the way it goes then who am I to complain :P

Reply:
I... personally *like* the extra whitespace (re: TABs ;p), but, yeah, house style rules take precedence ;p.